ApiDoctor.Validation/DocFile.cs

```diff
@@ -827,6 +823,19 @@ private void PostProcessEnums(List<EnumerationDefinition> foundEnums, List<Table
             // add all the enum values
             this.enums.AddRange(foundEnums.Where(e => !string.IsNullOrEmpty(e.MemberName) && !string.IsNullOrEmpty(e.TypeName)));
 
+            // if we thought it was a table of type EnumerationValues, it probably holds enum values. 
+            // throw error if member name is null which could mean a wrong column name
+            foreach (var enumType in foundEnums.Where(e => string.IsNullOrEmpty(e.MemberName) && !string.IsNullOrEmpty(e.TypeName))
+                .Select(x => new { x.TypeName, x.Namespace}).Distinct())
+            {
+                var possibleHeaderNames = this.Parent?.TableParserConfig?.TableDefinitions.Rules
+                    .Where(r => r.Type == "EnumerationDefinition").SelectMany(r => r.ColumnNames["memberName"]) .ToList();
+
+                issues.Error(ValidationErrorCode.ParameterParserError,
```

Comment:
This might need some testing, but I feel like it will clarify that enum tables must have the correct column name. I think it's the most confusing part of writing docs, that I noticed.

Reply:
I did some testing on graph docs and I had to have this [PR](https://github.com/microsoftgraph/microsoft-graph-docs/pull/11580) to fix the errors discovered from this update.